act-rules-format.bs

```diff
@@ -695,26 +695,29 @@ Examples in this section include:
   "@context": "context.json",
   "@type": "Assertion",
   "assertedBy": {
-    "@type": "foaf:Organization",
-    "@id": "_:myOrg"
-  }
+    "@type": "Organization",
+    "@id": "_:myOrg",
+    "title": "My Organization",
```

Comment:
Moved the organisation in, because the way it was added is incorrect. You could have created a `@graph` array with both the Assertion and the Organisation in as an alternative.